docs/schema-language.md

```diff
@@ -152,6 +160,23 @@ and create reference edges, while other generators mostly support only
 Nonetheless, decorations are first-class citizens of schema and thus are
 validated as well during archive opening.
 
+## Constant Referencing
+
+``@const( <name> )`` can be added to fields of a structure to indicate
+in which locations a constant can appear, e.g.:
+
+```cpp
+const u32 MY_CONST = 10;
+struct MyStruct {
+    @const( MY_CONST )
```

Comment:
This is only for documentation purposes (and dot file) atm, right?

Reply:
It also affects some schemas:
The constants are only serialized into the schema of resources using it (instead of the full archive)